packages/subgraph/src/helpers/index.ts

```diff
@@ -10,7 +10,13 @@ import {
 } from '../../generated/schema'
 import { STATUS_ACTIVE, STATUS_ACTIVE_NUM } from '../statuses'
 
+export const ZERO_ADDRESS = Address.fromString('0x0000000000000000000000000000000000000000')
+
 /// /// Token Entity //////
+export function isOrgToken(token: TokenEntity | null): boolean {
```

Comment:
What you think of using a more descriptive name for the function `isOrgToken`? Took me a bit of time to understand what was going on.

Reply:
What do you think about `checkForOrganization()` or `checkForOrg()` ?

Reply:
Perhaps something like `tokenHasOrg`?

Reply:
Yes, I like the last one that Fabri suggested.